src/Pages/Home/components/Workspace.jsx

```diff
@@ -0,0 +1,19 @@
+import WorkSpaceCard from "./WorkSpaceCard";
+
+const Workspace = ({ title, icon, data }) => {
+  return (
+    <section className="mb-20">
+      <h1 className="text-xl font-bold flex items-center gap-3">
+        {icon}
+        {title}
+      </h1>
+
+      <div className="my-4 flex justify-start items-center gap-4 flex-wrap">
+        {data.map((item) => {
```

Comment:
please add a check here for length/null of variable `data` in case and empty array is passed or it is not passed at all.